chapters/refactoringIdeas.typ

```diff
@@ -1,5 +1,6 @@
 = Refactoring Ideas <refactoring_ideas>
 // COR Per se wäre es wohl spannend gewesen alle Ideen zu beschreiben, auch wenn sie keine Refactorings sind oder nicht umsetzbar sind.
+// Jeremy Did we have other ideas?
```

Comment:
Ja, können wir kurz anschauen am Montag.
Z.B. extract to concept

Reply:
maches suscht imne neue PR